CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ cmake_minimum_required(VERSION 3.8)
 
 project(cmcstl2 CXX)
 
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
This `CMakeLists.txt` eschews "global" properties and attaches properties to targets. We should bump `cxx_std_17` on line 37 to `cxx_std_20` instead.